webapp/src/components/shared/BundledIcons.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    Add20Filled,
+    Add20Regular,
+    AppsAddIn24Filled,
+    AppsAddIn24Regular,
+    ArrowDownload16Filled,
+    ArrowDownload16Regular,
+    BotAdd20Filled,
+    BotAdd20Regular,
+    Checkmark20Filled,
+    Checkmark20Regular,
+    Delete16Filled,
+    Delete16Regular,
+    Dismiss16Filled,
+    Dismiss16Regular,
+    Dismiss20Filled,
+    Dismiss20Regular,
+    EditFilled,
+    EditRegular,
+    Filter20Filled,
+    Filter20Regular,
+    Info16Filled,
+    Info16Regular,
+    Share20Filled,
+    Share20Regular,
+    ThumbDislike16Filled,
+    ThumbDislike16Regular,
+    ThumbLike16Filled,
+    ThumbLike16Regular,
+    bundleIcon,
+} from '@fluentui/react-icons';
```

Comment:
Do we use all of these??

Reply:
Yeah these are all currently used, so easiest to port them all over now